### ISSUE TITLE
onnx infer

### ISSUE THREAD

**Issue by 1235555555shan** [open-mmlab/mmpose#1509]
Hello, when the human bone point model was converted into onNX model, the points generated during the loading prediction did not match the previous one. The image 1*3*256*192 was also loaded according to the previous preprocessing. Does anyone know that
`from mmcv.parallel import collate, scatter
    from mmpose.datasets.pipelines import Compose

    config = {
        'modelPaths': 'rsn50.onnx',
        'filenames': '1.jpg'
    }
    frame = cv2.imread(config['filenames'])
    x, y, w, h = 366, 367, 423, 1208
    bbox = [x, y, w, h]
    # ----------------------------
    flip_pairs = [[1, 2], [3, 4], [5, 6], [7, 8], [9, 10], [11, 12],
                  [13, 14], [15, 16]]
    for transform in val_pipeline:
        if transform['type'] == 'TopDownGetBboxCenterScale':
            has_bbox_xywh2cs = True
            break
    test_pipeline = Compose(val_pipeline)
    batch_data = []

    data = {
        'bbox': bbox,
        'bbox_score': bbox[4] if len(bbox) == 5 else 1,
        'bbox_id': 0,  # need to be assigned if batch_size > 1
        'dataset': 'coco',
        'joints_3d': np.zeros((17, 3), dtype=np.float32),
        'joints_3d_visible': np.zeros((17, 3), dtype=np.float32),
        'rotation': 0,
        'ann_info': {'image_size': np.array([192, 256]), 'num_joints': 17, 'flip_pairs': flip_pairs }
    }
    data['img'] = frame
    data = test_pipeline(data)
    batch_data = []
    batch_data.append(data)
    batch_data = collate(batch_data, samples_per_gpu=len(batch_data))
    # batch_data = scatter(batch_data, ['cuda:0'])[0]

    image_size = [192, 256]
    aspect_ratio = image_size[0] / image_size[1]
    center, scale = bbox_xywh2cs(
        bbox,
        aspect_ratio=aspect_ratio,
        padding=1.25,
        pixel_std=200.0)

    # rotation = 0
    # trans = utils.get_affine_transform(center, scale, rotation, input_size)
    # img = cv2.warpAffine(frame, trans, (int(input_size[0]), int(input_size[1])), flags=cv2.INTER_LINEAR)
    # cv2.imwrite('showimg2.jpg', img)
    # mean = [0.456, 0.456, 0.456]
    # std = [0.224, 0.224, 0.224]
    # img = F.to_tensor(img)
    # normalize_img = F.normalize(img, mean=mean, std=std)
    # normalize_img = np.reshape(normalize_img, [-1, 3, input_size[1], input_size[0]])
    # ort_inputs = normalize_img.numpy().astype("float32")

    heatmap = inference_with_onnx(config, batch_data['img'].numpy())[0]  # infer   inference_with_onnx
    score = np.ones(1)
    preds, maxvals = keypoints_from_heatmaps(
        heatmap,
        center.reshape(1, 2),
        scale.reshape(1, 2),
        unbiased= False,
        post_process='megvii',
        kernel=5,
        valid_radius_factor= 0.0546875,
        use_udp= False,
        target_type='GaussianHeatmap')

    res = preds.tolist()
    img = vis_pose(frame, res[0])
    cv2.imwrite('test4.jpg', img)`
